armi/tests/detailedAxialExpansion/refSmallReactorBase.yaml

```diff
@@ -160,15 +160,15 @@ blocks:
         clad: &component_plenum_clad
             shape: Circle
             material: HT9
-            Tinput: 25.0
+            Tinput: 27.548
             Thot: 470.0
             id: 1.0
             mult: 169.0
             od: 1.09
         wire: &component_plenum_wire
             shape: Helix
             material: HT9
-            Tinput: 25.0
+            Tinput: 27.548
```

Comment:
Can you explain in the PR description why these are changing? I assume that this is because we have `inputHeightsConsideredHot` set to `True` for the tests? Not clear though.

Reply:
The reference temperature of HT9 for thermal expansion is 27.548C. The test to check tohat density is conserved only works if Tinput is equal to the reference temperature for thermal expanison.

Reply:
In line with @jakehader 's comment:

@mgjarrett - for our future selves I think it would be good to leave a comment in this PR on precisely why you/we are making the changes to these temperatures and to armi/materials/zr.py (especially the latter). 

I could easily see us coming back a few months later and being totally stumped as to why these temperatures were all changed.

Reply:
@albeanth are you looking for a comment directly in the yaml file? Or just include it in the commit message?

Reply:
The commit message would be fine, but I was thinking about just a general comment in this PR. That way if someone in the future does a git blame and comes back to this PR it will be (somewhat) front and center.

I suppose a comment in the yaml would be ok too - maybe at the very top right off the bat? Not sure where it's best to include it. I'll leave it to your discretion.